lib/mixin.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+function arrayUnion(a, b) {
+  let map = {};
+  let i = 0;
+  for (; i < a.length; i++) {
+    map[a[i]] = true;
+  }
+  for (i = 0; i < b.length; i++) {
+    map[b[i]] = true;
+  }
+  return Object.keys(map);
+}
+
+module.exports = function mixin(ext, parent) {
+  arrayUnion(Object.keys(ext), Object.keys(parent)).forEach(function(key) {
+    const extValue = ext[key];
+    const parentValue = parent[key];
+    
+    // If both the object and the parent define the same function
+    if (typeof extValue === 'function' && typeof parentValue === 'function') {
+      if (extValue.toString().indexOf('_super') > -1) {
+        ext[key] = function mixinSuperWrapper() {
+          this._super = parent;
+          return extValue.apply(this, arguments);
+        };
+      }
+    } else if (typeof parentValue !== 'undefined' && !extValue) {
```

Comment:
What if `extValue` is falsy, I believe it should still trump that of parentValue.